src/commands/config/create.ts

```diff
@@ -0,0 +1,88 @@
+import fs from "fs";
+import { error, info, success } from "signale";
+import { Argv } from "yargs";
+import { deployDocumentStore } from "../../implementations/deploy/document-store/document-store";
+import { deployTokenRegistry } from "../../implementations/deploy/token-registry/token-registry";
+import { readFile } from "../../implementations/utils/disk";
+import { create as CreateWallet } from "../../implementations/wallet/create";
```

Comment:
bad case => createWallet, same everywhere

Reply:
ok will update it